src/commands/Games/higherLower.ts

```diff
@@ -0,0 +1,246 @@
+import { SkyraCommand } from '../../lib/structures/SkyraCommand';
+import { CommandStore, KlasaMessage } from 'klasa';
+import { MessageEmbed } from 'discord.js';
+import { LongLivingReactionCollector, LLRCData } from '../../lib/util/LongLivingReactionCollector';
+import { UserSettings } from '../../lib/types/settings/UserSettings';
+import { Emojis } from '../../lib/util/constants';
+import { getColor } from '../../lib/util/util';
+
+enum ReactionEmoji {
+	HIGHER = '⬆',
+	LOWER = '⬇',
+	CANCEL = '❌', // TODO: Replace with <:redCross:637706251257511973>
+	OK = '✔️', // TODO: Replace with <:greenTick:637706251253317669>
+	CASHOUT = '💰'
+}
+
+const enum EndingAction {
+	PLAY,
+	STOP,
+	TIMEOUT
+}
+
+export default class extends SkyraCommand {
+
+	public constructor(store: CommandStore, file: string[], directory: string) {
+		super(store, file, directory, {
+			aliases: ['hilo', 'higherlower', 'hl'],
+			bucket: 2,
+			cooldown: 7,
+			description: `Play the Higher Lower game against me`,
+			extendedHelp: `as well as this`,
+			requiredPermissions: ['ADD_REACTIONS', 'MANAGE_MESSAGES', 'EMBED_LINKS'],
+			runIn: ['text'],
+			usage: '<50|100|200|500|1000|2000|5000|10000>'
+		});
+	}
+
+	public async run(message: KlasaMessage, [text]: [string]) {
+		const wager = Number(text);
+		const balance = message.author.settings.get(UserSettings.Money);
+		if (wager > balance) throw message.language.get('COMMAND_SLOTMACHINES_MONEY', balance);
```

Comment:
Please rebase with master. This has been renamed to `GAMES_NOT_ENOUGH_MONEY` (or something like that)